README.md

```diff
@@ -4,44 +4,104 @@
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
 
+## Introduction
+
+### Purpose
+
+_Growth, Maturity and Decline_ is a working group focused on the lifecycle of open source projects. We do not valence the stages, but seek to build awareness of those stages among consumers of these metrics. Decline can be ok when a project's utility is waning; or terrible if 20% of the entire internet relies on that project. Context matters, and we are not the arbiters of your context. The "valence" of "good/bad" emerges from use cases and YOUR context.
+
+### Goal
+Currently, the goals are structured in the following focus areas : [Issue Resolution](./focus_areas/issue_resolution.md), [Code Development](./focus_areas/code_development.md) and [Community Growth](./focus_areas/community_growth.md).
+
+### Outcome
+
+### Process
+For each area of interest, we're following the [goal-question-metric](https://en.wikipedia.org/wiki/GQM) methodology defining questions and metrics that help to answer them.
+
 ## Table of Contents
+
 - [Introduction](#introduction)
-- [Contributing](#contributing)
+- [Background](#background)
+- [Roadmap for 2019](#roadmap-for-2019)
 - [Metrics](#metrics)
 - [Usage](#usage)
-- [Maintainers](#maintainers)
+- [Implementations](#implementations)
+- [Software work](#software-work)
+- [Use Cases](#use-cases)
+- [Related Works](#related-works)
+- [Contributing](#contributing)
+- [Communications](#communications)
+- [Contributors](#contributors)
 - [License](#license)
 
+## Background
 
+### Problem Statement
 
-## Introduction
+To be updated.
 
-_Growth, Maturity and Decline_ is a working group focused on the lifecycle of open source projects. We do not valence the stages, but seek to build awareness of those stages among consumers of these metrics. Decline can be ok when a project's utility is waning; or terrible if 20% of the entire internet relies on that project. Context matters, and we are not the arbiters of your context. The "valence" of "good/bad" emerges from use cases and YOUR context.
+### Opportunity
 
-## Contributing
+To be updated
+
+### Work To Date
+
+To be updated.
+
+## Roadmap for 2019
+
+Our key focus areas for 2019 are summarized [here](./ROADMAP.md).
+
+## Metrics
+
+The metrics dealt with in this focus group are organized in focus areas:
+
+* Growth Maturity and Decline
+  * [Code Development](./focus_areas/code_development.md)
```

Comment:
I'm not sure if it's better to put the full link instead of the dot (https://github.com/chaoss/wg-gmd/blob/master/focus_areas/code_development.md instead of ./focus_areas/code_development.md)

Reply:
I would rectify the PR to include the complete links!

Reply:
> I'm not sure if it's better to put the full link instead of the dot (https://github.com/chaoss/wg-gmd/blob/master/focus_areas/code_development.md instead of ./focus_areas/code_development.md)
> 
> This comment applies to most links in the PR

I think it is better to have the relative link, since that way it will always point to the right place, even if the repo changes in name (which is something that could happen during the next few days). Additionally, I would skip `./`, since the relative link would work also without it.

Reply:
Ok. @jgbarah, will do the needful!